spring-cloud-dataflow-rest-client/pom.xml

```diff
@@ -33,6 +33,10 @@
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
If this dependency is just for test only, can we just restrict it for tests alone?